docs-markdown/src/test/suite/controllers/quick-pick-menu-controller.test.ts

```diff
@@ -73,6 +73,14 @@ const context: ExtensionContext = {
 suite('Quick Pick Menu Controller', () => {
 	suiteSetup(() => {
 		sinon.stub(telemetry, 'sendTelemetryData');
+		sinon.stub(workspace, 'getConfiguration').returns({
```

Comment:
There is a convention throughout the unit tests to override methods such as `getConfiguration()`.  However, this can result in unexpected behavior if someone is writing a new test and expecting the default behavior.  The use case I had was writing a new test case and expecting the configurations to be returned.  These overrides prevented my test from receiving the valid configuration.

For example, I have added config property 

```
{
  "markdown.metadataNag": true
}
```
to path `docs-markdown/src/test/data/repo/.vscode/settings.json`.  When I tried to load this using `workspace.getConfiguration()` it returns undefined
<img width="509" alt="Screen Shot 2020-06-29 at 5 41 52 PM" src="https://user-images.githubusercontent.com/46545485/86071571-76a7de00-ba34-11ea-99c6-0b23d85fbeb4.png">

By converting these overrides to stub we get the default behavior in new tests where it is defined.   You can see below how the property returns. This also allows us to not repeat ourselves with code in the unit tests.

<img width="363" alt="Screen Shot 2020-06-29 at 5 51 58 PM" src="https://user-images.githubusercontent.com/46545485/86071559-6bed4900-ba34-11ea-9db9-c791209d2858.png">